lib/generators/pageflow/install/install_generator.rb

```diff
@@ -18,6 +18,7 @@ def generate_all
         invoke 'pageflow:user'
         invoke 'pageflow:seeds'
         invoke 'pageflow:active_admin_initializer'
+        invoke 'pageflow:not_found'
```

Comment:
I think `pageflow:error_pages` would be a bit more descriptive and would also allow to include files for other status codes should we ever need them.